src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -173,7 +174,11 @@ private String getResponse(String url, String fieldParam) {
       return null;
     }
     JSONObject json = new JSONObject(responseBody.toString());
-    String res = json.getJSONArray(ITEM_PARAMETER).getJSONObject(0).get(fieldParam).toString();
+    JSONArray items = json.getJSONArray(ITEM_PARAMETER);
+    if (items == null || items.length() == 0) {
+      return null;
+    }
+    String res = items.getJSONObject(0).get(fieldParam).toString();
```

Comment:
nit: change to "result" or something more readable

Reply:
Unresolved?

Reply:
uh alright, i just feel res is somewhat common